src/frontend/src/handler/create_mv.rs

```diff
@@ -79,6 +79,7 @@ pub fn gen_create_mv_plan(
     query: Query,
     name: ObjectName,
     columns: Vec<Ident>,
+    emit_mode: Option<EmitMode>,
```

Comment:
qmaybe adding it on the planRoot is better? It is more like a "query's property". I think we will support the EOWC sink soon.

Reply:
Looks like it's more like `WITH`. Maybe we can refactor it later when needed😇